core/src/main/java/com/orange/ouds/core/theme/OudsTheme.kt

```diff
@@ -123,19 +123,19 @@ object OudsTheme {
 /**
  * [OudsTheme] is the theme to apply to your screens in a Jetpack Compose application. Use it at the top of
  * your application in replacement of the `MaterialTheme`.
- * Cause OUDS supports multi-theme, you should pass a [themeContract] as theme configuration to use an OUDS supported theme.
+ * Cause OUDS supports multi-theme, you should pass the OUDS supported [theme] used by your application.
  *
- * @param themeContract Theme contract which contain the configuration of the OudsTheme: colors, typography...
+ * @param theme Theme to apply to your application. It must implement [OudsThemeContract] (e.g. OrangeTheme, Sosh Theme, ...)
```

Comment:
`Sosh Theme` contains a space whereas this is not the case with `OrangeTheme`. I don't know which one is the best (obviously it depends if you talk about the classe or not) but we should homogenize both.